models/comments.models.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection.js")
+
+exports.selectCommentsByArticleID = (article_id) => {
+    return db.query(`
+    SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at desc
+    `, [article_id])
+    .then((res) => {
```

Comment:
this variable in the callback could better describe what the query is getting, I would call it comments here